src/snackbar/snackbar.spec.tsx

```diff
@@ -123,17 +118,14 @@ describe('SnackbarQueue', () => {
       ]
     });
 
-    await wait(500);
-
-    expect(el.html().includes('myNotificationTitle1')).toBe(true);
-    expect(el.html().includes('myNotificationBody1')).toBe(true);
+    waitFor(() => {
+      expect(screen.getByText('myNotificationTitle1')).toBeInTheDocument();
+      expect(screen.getByText('myNotificationBody1')).toBeInTheDocument();
+    });
```

Comment:
What about the second notification. No verify on that?

Reply:
It's right below